authorizations.md

```diff
@@ -0,0 +1,92 @@
+* scope: link to the object scope where this authorization apply. Not limited to DSpace objects, see [features endpoint](features.md) for more details
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated and the policy is not related to the ANONYMOUS user (eperson null)
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators and the user mentioned in the authorization can access
+* 404 Not found - if the authorization doesn't exist (or was already deleted)
+
+### Search methods
+#### resource
+**/api/authz/authorizations/search/scope?uri=<:uri>[&eperson=<:uuid>&feature=<:string>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uri: mandatory, the scope to use for the authorization check. The full URI of the rest resource must be specified, i.e. https://{dspace.url}/api/core/community/{uuid}
+* eperson: optional, the uuid of the eperson to evaluate for authorization. If not specified authorization of anonymous users will be returned
+* feature: optional, limit the returned authorization to the specified feature (this provide an alternative to codify the authorization id rule on the client side)
+
+It returns the list of matching authorizations. Please note that all the matching authorizations available for the requested user will be returned including the one available to anonymous users. There is no need on the client side to combine the authorizations of the current, logged-in, user with the authorizations of anonymous users. 
+
+Return codes:
+* 200 OK - if the operation succeed
+* 400 Bad Request - if the uuid parameter is missing or invalid
+* 401 Forbidden - if you are not authenticated and an eperson is specified
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators and the user specified in the eperson parameter can access
+
+## Other HTTP verbs than GET
+No other verbs are supported right now. Authorizations cannot be grant or revoke directly, they are generated by the system via interaction with other endpoint such as the resourcepolicies, the submission, the workflow, etc.
+
+## Linked entities
+### EPerson
+**/api/authz/authorizations/<:string>/eperson**
+
+Provide access to the eperson linked by this authorization
+
+Return codes:
+* 200 Ok - if the operation succeed and an eperson is set for this policy
+* 204 No content - if the authorization is related to anonymous users
+* 401 Forbidden - if you are not authenticated and the authorization is not related to the Anonymous user
+* 403 Unauthorized - if you are not logged in with sufficient permissions. See the requirement in the GET Single Authorization endpoint
+* 404 Not found - if the authorization doesn't exist
+
+### Scope
+**/api/authz/authorizations/<:string>/scope**
+
+Provide access to the resource where the authorization is scoped.
```

Comment:
Again, the word `scope` here is a bit confusing, as this is the *object* or resource that the authorization applies to.

Reply:
done